lib/compat/wordpress-6.2/get-global-styles-and-settings.php

```diff
@@ -255,3 +255,45 @@ function _gutenberg_add_non_persistent_theme_json_cache_group() {
 	wp_cache_add_non_persistent_groups( 'theme_json' );
 }
 add_action( 'plugins_loaded', '_gutenberg_add_non_persistent_theme_json_cache_group' );
+
+
+/**
+ * Returns a string containing the SVGs to be referenced as filters (duotone).
+ *
+ * @since 5.9.1
+ *
+ * @return string
+ */
+function gutenberg_get_global_styles_svg_filters() {
```

Comment:
This is just a copy of `gutenberg_get_global_styles_svg_filters` with the reference to the `WP_Theme_JSON_Resolver` class updated to `WP_Theme_JSON_Resolver_Gutenberg`.